src/compile/data/window.ts

```diff
@@ -59,6 +59,10 @@ export class WindowTransformNode extends DataFlowNode {
     return windowFieldDef.as || vgField(windowFieldDef);
   }
 
+  public hash() {
+    return hash(this.transform);
```

Comment:
Do you think it makes sense to have tests for these hash functions so we can read them in the diff and see whether they look good?

Reply:
Are you suggesting tests like
i) `hash(node) === hash(node.clone())`
or 
ii) `hash(node)` === `'__hash_value__'`
I see the benefit that *test ii* might provide, but the value will have to be updated each time we change things.
However, I think I should write some tests for the hash function itself with probable inputs in `util.test.ts`.

Reply:
Latter